modules/end-user-guide/partials/proc_building-an-image-for-an-ide-based-on-the-intellij-platform.adoc

```diff
@@ -0,0 +1,160 @@
+[id="building-an-image-for-an-ide-based-on-the-intellij-platform_{context}"]
+= Building an image for an IDE based on the IntelliJ Platform
+
+This procedure describes how to build an image for an IDE based on the IntelliJ Platform up to version 2020.3.
+
+.Prerequisites
+
+* The build host has at least 2 GB of available RAM.
+
+* The build host has a local installation of following tools:
```

Comment:
`installation` = the act of putting a program onto a computer
imho `local` (as in locally installed) is redundant and can be dropped if the location/host is specified (here `the build host`).

Reply:
fixed.